blazor/spreadsheet/potection.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Protect sheet in Blazor Spreadsheet Component | Syncfusion
+description: Checkout and learn here all about protect sheet in Syncfusion Blazor Spreadsheet component and more | Syncfusion.
+platform: Blazor
+control: Spreadsheet
+documentation: ug
+---
+
+# Protect sheet in Blazor Spreadsheet Component
+
+Sheet protection helps to prevent modification of data in the spreadsheet.
+
+## Protect Sheet
+
+The Protect Sheet feature restricts accidental changes such as editing, moving, or deleting data. Protection can be applied with or without a password.
+
+### Protect Settings in Protected Sheet
+
+By default, when a sheet is protected, actions such as selecting cells, formatting, inserting, sorting and filtering are disabled.
```

Comment:
This content is wrong. Please refer the following content,

By default, when a sheet is protected, most actions such as formatting, inserting, sorting, and filtering are restricted, while selecting cells remains allowed.